drivers/hmis/app/graphql/types/hmis_schema/enums/form_status.rb

```diff
@@ -0,0 +1,14 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Types
+  class HmisSchema::Enums::FormStatus < Types::BaseEnum
+    graphql_name 'FormStatus'
+
+    Hmis::Form::Definition::STATUSES.each_with_index do |status, index|
+      value status, description: "#{index + 1} - #{status.titleize}"
```

Comment:
What's the reasoning behind the numeric value in the description here?
The HUD enums have it because they are stored as integers, but in this case I don't see any relation to this number and the status.

Reply:
Good point, I probably just copied and pasted this from somewhere, but you're right, it is neater without the indexes! :)